motion_predict/src/predict_ctrv.cpp

```diff
@@ -61,9 +61,13 @@ CTRV_State buildCTRVState(const geometry_msgs::msg::Pose& pose, const geometry_m
   CTRV_State state;
   state.x = pose.position.x;
   state.y = pose.position.y;
-  state.yaw =
-      rpy[2] + std::get<0>(vel_angle_and_mag);  // The yaw is relative to the velocity vector so take the heading and
-                                                // add it to the angle of the velocity vector in the local frame
+  state.yaw = std::get<0>(vel_angle_and_mag); // Currently, object's linear velocity is already in map frame.
```

Comment:
`buildCTRVState` is called as part of the pipeline starting in carma-platform's `MotionComputationWorker::predictionLogic()` [here](synchronization_base_objects). I think it is fair for the `motion_predict` package to assume objects are provided in the `map` frame, but [1] we should document this in either the package's README or in the applicable function descriptions in the header file(s), and [2] I think we may want to consider a separate PR in carma-platform's `motion_computation` package to check in `MotionComputationWorker::predictionLogic` whether the provided object list is in the "map" frame, and log a warning statement (for now at least, in the future we could implement a conversion in this case) if it isn't, since it looks like the map frame is a requirement for our entire object tracking and motion prediction pipeline. Thoughts on this?